source/common/config/well_known_names.cc

```diff
@@ -106,6 +106,9 @@ TagNameValues::TagNameValues() {
 
   // mongo.(<stat_prefix>.)*
   addRegex(MONGO_PREFIX, "^mongo\\.((.*?)\\.)");
+
+  // http.[<stat_prefix>.]rds.(<route_config_name>.)<base_stat>
```

Comment:
Do we have any dot path stats under the route config today? Should we allow freedom to do this in the future?

Reply:
We don't currently afaict - https://www.envoyproxy.io/docs/envoy/latest/configuration/http_conn_man/rds.html.

FWIW, I basically just copied the regex for [HTTP_USER_AGENT](https://github.com/envoyproxy/envoy/blob/d075f6c02d83196364aab0aa53d78358b8fc2b31/source/common/config/well_known_names.cc#L65) just replacing "user_agent" with "rds"

Reply:
If you can file a followup issue related to the `.` in identifiers and these regexes, LGTM.

Reply:
Filed #6622